### ISSUE TITLE
/{key}.json URL mapping is insufficient for REST resources with Path variables.

### ISSUE THREAD

**Issue by nblair** [UW-Madison-DoIT/rest-proxy#9]
Example use case:

In my application (referred to with /app) want to proxy a REST API that provides the following resource:

> /star/api/v2/employee/UW111A111/timecards

I would add the following to "endpoint.properties"

```
star.uri=http://localhost:8080/star
star.username=...
star.password=...
```

I'd bind the Spring DispatcherServlet for this application to the (servlet context relative) url _/proxy_.

I want to be able to then access the aforementioned resource at the URL:

> /app/proxy/star/api/v2/employee/UW111A111/timecards

In the current state, however, the Controller provided by the library only exposes:

> /app/proxy/star.json

And gives me no way to remit additional path parameters through the proxy to the target REST API.

**Comment by nblair** [comment]
I should note that the variable part of that path is the "UW111A111" part.

**Comment by timlevett** [comment]
Cool. Seems like that would be really handy.

**Comment by timlevett** [comment]
Resolved by #10
